README.md

```diff
@@ -187,4 +187,7 @@ request dated after any votes resets those votes.
 If the rule-change is adopted, a player who can shall merge the pull-request
 in a timely fashion, which marks completion of the vote.
 
-
+**301** If a PR that outlines a rule-change (in accordance with other rules in
+this file) contains `y` commit hashes containing an English dictionary word of 
```

Comment:
'If a PR that mutates the rule set contains ...'
My point is that 'rule-change' can be interpreted as:
1. A change in the rule set.
2. A rule amendment.